html5/_main_toc.html.slim

```diff
@@ -20,6 +20,9 @@ nav.sidebar-nav
         li
           a.nav-link href="/latest/quick_start_with_kb_api.html"
             | Quick Start with the Kill Bill API
+        li
+          a.nav-link href="/latest/what_is_aviate.html"
+            | What is Aviate?                          
     li
```

Comment:
Should this be moved to the new Aviate section here?

<img width="233" alt="image" src="https://github.com/user-attachments/assets/a4cda767-5553-4246-b9dd-1b8e61cb5de8">

Reply:
Fixed in [03c6e0c](https://github.com/killbill/killbill-docs/pull/571/commits/03c6e0c32c27053a151a4c8a73f9e923fecea4c7)